packages/SidePanel/src/components/Header/Header.styles.js

```diff
@@ -0,0 +1,40 @@
+import { fontSize } from "@paprika/stylers/lib/helpers";
+import tokens from "@paprika/tokens/lib/tokens";
+import { css } from "styled-components";
+
+const kind = {
+  default: `background: ${tokens.color.white}; color: ${tokens.color.black};`,
+  primary: `background: ${tokens.color.purple}; color: ${tokens.color.white};`,
+};
+
+const space = Number.parseInt(tokens.space, 10);
+export const headerCSS = css`
+  border-bottom: 1px solid ${tokens.border.color};
+  box-sizing: border-box;
+  color: ;
```

Comment:
✂️

Reply:
thank you! you found the damn null that I was seeing in the dev console and couldn't found! this makes my soul rest...